integration/network/service_test.go

```diff
@@ -25,7 +25,9 @@ func delInterface(t *testing.T, ifName string) {
 }
 
 func TestDaemonRestartWithLiveRestore(t *testing.T) {
-	skip.If(t, testEnv.IsRemoteDaemon(), versions.LessThan(testEnv.DaemonAPIVersion(), "1.38"))
+	skip.If(t, testEnv.IsRemoteDaemon())
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.38"),
+		"testing new feature added after 1.37")
```

Comment:
@AntaresS you can keep it on same line

Reply:
Also, no need to add version in the message string, off-by-ones can be confusing.

Reply:
@tiborvass Done. It's exceeding the ruler on my IDE, so thought would be better to split up into two lines.